test/e2e/flowaggregator_test.go

```diff
@@ -987,79 +997,71 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 
 func checkRecordsForFlowsCollector(t *testing.T, data *TestData, srcIP, dstIP, srcPort string, isIPv6, isIntraNode, checkService, checkK8sNetworkPolicy, checkAntreaNetworkPolicy bool, bandwidthInMbps float64, labelFilter string) {
```

Comment:
any actual change in this function or is it just reformatting?

Reply:
Some changes are made:
1. remove` if strings.Contains(record, src) && strings.Contains(record, dst) && strings.Contains(record, srcPort)` as we already filter the record when retrieving the log from the IPFIX collector Pod.
2. We don't need to count dataRecordsCount as the length of recordSlices should be the total number of matched records.